features/support/world.rb

```diff
@@ -81,11 +81,16 @@ def conjur_authenticate_from_json(response_json)
     appliance_url = parse_json(response_json, 'credentials/appliance_url')
     login = parse_json(response_json, 'credentials/authn_login')
     api_key = parse_json(response_json, 'credentials/authn_api_key')
+    version = parse_json(response_json, 'credentials/version')
+    ssl_cert = parse_json(response_json, 'credentials/ssl_certificate')
 
     Conjur.with_configuration Conjur::Configuration.new(
         account: account,
-        appliance_url: appliance_url
+        appliance_url: appliance_url,
+        ssl_certificate: ssl_cert,
```

Comment:
Not going to change this to use `Conjur.configure do |config|`?

Reply:
I didn't in this update, because this worked fine and has the value of being a temporary config update for the block below. I wasn't sure how changing it generally would affect other tests.

If you find it's needed to change this for your tests, I'm happy to consider that in your PR - but an update isn't needed for these changes.